src/main/java/mate/academy/dao/MovieSessionDao.java

```diff
@@ -6,9 +6,9 @@
 import mate.academy.model.MovieSession;
 
 public interface MovieSessionDao {
-    List<MovieSession> findAvailableSessions(Long movieId, LocalDate date);
+    MovieSession add(MovieSession movieSession);
 
     Optional<MovieSession> get(Long id);
 
-    MovieSession add(MovieSession session);
+    List<MovieSession> findAvailableSessions(Long movieId, LocalDate date);
```

Comment:
why you chenge this?

Reply:
I copied files from preveous homwork, i guess methods ther were in a different order